src/app/home/problem-list/user-profile/user-profile.component.html

```diff
@@ -0,0 +1,90 @@
+<div class="containerX">
+  <div class="container">
+    <div fxLayout="row" fxLayoutAlign="space-around center">
+      <h1>Welcome {{ userInfo.displayName }} !</h1>
+    </div>
+    <mat-divider></mat-divider>
+  </div>
+  <div class="container">
+    <div fxLayout="row" fxLayoutAlign="space-around center">
+      <mat-card>
+        <img [src]="userInfo.photoUrl" />
+      </mat-card>
```

Comment:
My image overflows the container as there is no constraint on it. I would recommend removing the card and changing to just `<img [src]="userInfo.photoUrl">` and adding a style of `max-width: 10rem; border-radius: 50%;` to make it a small round image. Also there is a `mat-card-image` directive, if you still want the card.


Currently, it looks like this
<img width="1365" alt="Screenshot 2021-05-20 at 12 16 05 PM" src="https://user-images.githubusercontent.com/6594255/119036061-2d6ebe80-b965-11eb-940b-94f0ab1b7849.png">